### ISSUE TITLE
download media with search results?

### ISSUE THREAD

**Issue by AJLinn** [ArctosDB/arctos#1552]
I just discovered the incredibly valuable ability to include media in my search results page (thank you!) but I'm wondering if it's possible to include those thumbnails when I download the results rather than just the URL to the thumbnail? I currently have my preferred file format listed as CSV and I get just a URL when I include the media in the download. 

If I select a different format for downloading, will the media image come with it? OR am I going to have to learn how to do a report... 😢

**Comment by dustymc** [comment]
I'm not aware of anything suitable for that, but I'm open to ideas. CSV is as far as I know text-only. Do you just want the preview, or do you want the media objects?

**Comment by AJLinn** [comment]
Just the thumbnail preview would be perfect. 
Here's an example of a report the Anchorage Museum recently sent me. I'd love to be able to produce something similar.
![mediareport_anchss](https://user-images.githubusercontent.com/17605945/41299982-2366135e-6e11-11e8-8ceb-300948f10bd1.jpeg)

But it could literally just be a regular spreadsheet-style of output with the media preview as the first column.

**Comment by dustymc** [comment]
>*the* media preview

What if there are 50 of them?

CF can theoretically create .xls files, and .xls files can theoretically support embedded images, so "technically plausible" anyway. I'm not sure we're going to be able to automagically squish the potential data in Arctos down to anything quite that nice. 

Probably not pretty enough, but maybe you can print to PDF?

Or just send them an Archive URL?

**Comment by AJLinn** [comment]
> What if there are 50 of them?

yeah, that's a problem... just the first one is enough (or is there a way to mark one image as the first to show up when there are many?)

> CF can theoretically create .xls files, and .xls files can theoretically support embedded images, so "technically plausible" anyway. I'm not sure we're going to be able to automagically squish the potential data in Arctos down to anything quite that nice.

I'd be willing to see what we could manage.

> Probably not pretty enough, but maybe you can print to PDF?

Exactly what I've been doing now that I know I can show the media preview in my search results.

> Or just send them an Archive URL?

This is my standard process. I like an output like this when I do loans (since I haven't figured out the reports yet) or when I send a student out to retrieve a bunch of items from the range and they don't have access the laptop.

**Comment by Jegelewicz** [comment]
> 

    Or just send them an Archive URL?

This is my standard process. I like an output like this when I do loans (since I haven't figured out the reports yet) or when I send a student out to retrieve a bunch of items from the range and they don't have access the laptop.

Would a printable webpage work? Is that even a thing?  I'm thinking about when I make a flight reservation and I can print my itinerary from the web. I end up with a PDF that often includes pictures and other stuff....

**Comment by dustymc** [comment]
>printable webpage

Yea, it's just CSS. The big difference is that your flight itinerary is very predictable, while Arctos specimen results is somewhere between one row and a half-dozen columns and a hundred thousands rows and 207 (and constantly growing) columns, and the content is wildly variable (somewhere between nothing and 4000 characters). I'm not sure how realistic printing that is, but maybe there's some sort of "works for most users most of the time" solution.

**Comment by dustymc** [comment]
Tabling - AFAIK this isn't possible with our current tech stack.
